src/include/txn_inline.h

```diff
@@ -1383,7 +1383,9 @@ __wt_txn_read(
 
     /* If there's no visible update in the update chain or ondisk, check the history store file. */
     if (F_ISSET(S2C(session), WT_CONN_HS_OPEN) && !F_ISSET(session->dhandle, WT_DHANDLE_HS)) {
-        __wt_timing_stress(session, WT_TIMING_STRESS_HS_SEARCH, NULL);
+        /* Sleep randomly on every 100th iteration when history store search is enabled. */
```

Comment:
I think it would be better to add a comment that explains why the code is this way now. I'd suggest somethong along those lines:
```suggestion
        /* Stressing this path may slow down the system too much. When enabled, only sleep every 100th iteration to mitigate the impact. */
```

Reply:
I have rephrased the comment here d61076037e793d8c516a8ff4a9d56c1843fe1f58